README.md

```diff
@@ -49,13 +49,26 @@ use Request;
 
 class ApiController extends Controller
 {
-    public function handleRequest($modelName, $id = null)
+    /**
+     * Create handler name from request name
+     *
+     * @param $requestedResource string The name of the model (in plural)
```

Comment:
Type type should come before the name, see also http://phpdoc.org/docs/latest/references/phpdoc/tags/param.html